geopandas/geoseries.py

```diff
@@ -168,12 +152,12 @@ def __new__(cls, data=None, index=None, crs=None, **kwargs):
                     values = data.blocks[0].values
                     block = ExtensionBlock(values, slice(0, len(values), 1), ndim=1)
                     data = SingleBlockManager([block], data.axes[0], fastpath=True)
-                self = super(GeoSeries, cls).__new__(cls)
-                super(GeoSeries, self).__init__(data, index=index, **kwargs)
-                self.crs = getattr(self.values, "crs", crs)
-                return self
-            warnings.warn(_SERIES_WARNING_MSG, FutureWarning, stacklevel=2)
-            return Series(data, index=index, **kwargs)
+                    crs = getattr(values, "crs", crs)
```

Comment:
I think this is all that's needed here now, without the extra super() calls. 
`crs = getattr(values, "crs", crs)` might be superfluous, on the 37-mininimal env using pandas 0.25 only 3 tests go through this path and none require the crs = line, haven't exactly delved into that though. 
(test_pandas_methods.py::test_apply_loc_len1, and test_geoseries.py:test_from_wkb/wkt_series_with_index

Reply:
I would also think this line is no longer needed (now we store the crs on the GeometryArray level, and the `values` here is such an array. So if it has a crs, that will already be used)